packages/manager/modules/billing/src/components/service-status/index.js

```diff
@@ -0,0 +1,20 @@
+import 'angular-translate';
+import '@ovh-ux/ng-translate-async-loader';
+import '@ovh-ux/ui-kit';
+import ovhManagerCore from '@ovh-ux/manager-core';
+
+import component from './service-status.component';
+
+const moduleName = 'ovhManagerBillingServiceStatus';
+
+angular
```

Comment:
👍  great idea. Can this be a common component? Status is used in many places, almost in all Datagrid and dashboard pages.

Reply:
@radireddy This is why it has been extracted to the `@ovh-ux/manager-billing` module as currently this module exposes utilities for service billing handling (like `service-actions` component used across hub and Dedicated)

Reply:
Ok got it. What I mean is to use this status component in any Manager app/module across all Manager universal (cloud, server, dedicated, web, telco, etc). I was thinking this can be only used in dedicated/billing.